econml/utilities.py

```diff
@@ -1391,6 +1393,91 @@ def predict(self, X):
         return self._clf.predict_proba(X)[:, 1:]
 
 
+class _TransformerWrapper:
+    """Wrapper that takes a featurizer as input and adds jacobian calculation functionality"""
+
+    def __init__(self, featurizer):
+        pass
+        self.featurizer = featurizer
+
+    def fit(self, X):
+        return self.featurizer.fit(X)
+
+    def transform(self, X):
+        return self.featurizer.transform(X)
+
+    def fit_transform(self, X):
+        return self.featurizer.fit_transform(X)
+
+    def get_feature_names(self, X):
+        if hasattr(self.featurizer, 'get_feature_names'):
+            return self.featurizer.get_feature_names(X)
+
+        return []
+
+    def jac(self, X):
+        if hasattr(self.featurizer, 'jac'):
+            return self.featurizer.jac(X)
+        elif (isinstance(self.featurizer, PolynomialFeatures)):
+            self.featurizer.fit(X)
```

Comment:
Is there a contract that `jac` should only be called after `fit` has already been called?  (I think this would make sense.)  If so - document that on the jac method and then drop the fit call here.

Reply:
jac would only be called when someone is trying to calculate marginal effect estimates or intervals, which should be done after fit time. Will remove the featurizer.fit call here